src/codeflare_sdk/cluster/cluster.py

```diff
@@ -115,6 +119,9 @@ def up(self):
         Applies the AppWrapper yaml, pushing the resource request onto
         the MCAD queue.
         """
+        if self.app_wrapper_yaml is None:
+            print("Error putting up RayCluster: AppWrapper yaml not generated")
```

Comment:
Can we replace this error statement with:
```
self.app_wrapper_yaml = self.create_app_wrapper()
self.app_wrapper_name = self.app_wrapper_yaml.split(".")[0]
```

This way we just write it if it does not exists?